EIPS/eip-XXXX.md

```diff
@@ -0,0 +1,177 @@
+---
+Eip: XXXX *Number to be determined by Editor*
+Title: ERC Token in Multiple Languages
+Author: Jeremy Akatsa (@jeyakatsa)
+Status: Draft
+Type: Standards Track
+Category: ERC
+Created: 1/8/2022
+---
+
+## Simple Summary
+An ERC token/currency capable of being created in multiple languages (Java, Rust, Python, Go, etc).
```

Comment:
I think your simple summary is a bit too vague. Keep it short, but try to be a bit more specific, and touch on exactly what you want to accomplish.